crates/wallets/Cargo.toml

```diff
@@ -27,15 +27,9 @@ alloy-dyn-abi.workspace = true
 
 # aws-kms
 alloy-signer-aws = { workspace = true, features = ["eip712"], optional = true }
-aws-config = { workspace = true, default-features = true, optional = true }
-aws-sdk-kms = { workspace = true, default-features = false, optional = true }
 
 # gcp-kms
 alloy-signer-gcp = { workspace = true, features = ["eip712"], optional = true }
-gcloud-sdk = { version = "0.27.3", features = [
-  "google-cloud-kms-v1",
-  "google-longrunning",
-], optional = true }
```

Comment:
ref for other reviewers, this is covered in https://github.com/alloy-rs/alloy/blob/1c246a7e60d709edfb424b2af12674dadfecc280/crates/signer-gcp/Cargo.toml#L32-L35